### ISSUE TITLE
final_workflow_outputs_dir does not appear on output

### ISSUE THREAD

**Issue by Coppini** [broadinstitute/cromwell#6222]
When running a workflow on Cromwell using the following option 

```
{
    "final_workflow_outputs_dir" : "gs://my-bucket/my-dir"
}
```

I would expect the outputs of the cromwell job to reflect the final output dir. In fact (at least when running on Google Cloud), the outputs still appear with the path to my temporary directory (set on my backend `root`) .

The files are still copied and will show up on my `final_workflow_outputs_dir`, appearing both in the temp and final buckets, but when looking at the outputs on `{cromwell-url}/api/workflows/v1/{workflow_id}/outputs`, it will show the paths to the temporary bucket.

[Jira issue](https://broadworkbench.atlassian.net/browse/CROM-6711)

**Comment by aednichols** [comment]
Agreed, I was not aware of this story when working on that feature. Thanks for pointing it out, I will close this issue.
